corehq/apps/hqadmin/views.py

```diff
@@ -440,6 +447,18 @@ def get_context_data(self, **kwargs):
         return context
 
 
+class DomainAdminRestoreView(AdminRestoreView):
+    urlname = 'domain_admin_restore'
+
+    def dispatch(self, request, *args, **kwargs):
+        return TemplateView.dispatch(self, request, *args, **kwargs)
```

Comment:
I needed to skip the parent dispatch function, which is why im not just using `super` because it is restricted only to superusers and this endpoint is not. I don't love this solution but it seemed the best available.